mkosi

```diff
@@ -176,7 +176,7 @@ def image_size(args):
 def disable_cow(path):
     """Disable copy-on-write if applicable on filesystem"""
 
-    subprocess.run(["chattr", "+C", path], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=False)
+    subprocess.run(["chattr", "+C", path], stdout=subprocess.DEVNULL, check=False)
```

Comment:
this is swallowed for a reason: "+C" is not available on anything but btrfs at the moment, and we really don't want to see the error about it, as it's an optimization only anyway